.storybook/main.js

```diff
@@ -1,92 +1,145 @@
+    '../projects/canopy/src/lib/hero/docs/hero.stories.ts',
+    '../projects/canopy/src/lib/hero-img/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/hero-img/docs/hero-img.stories.ts',
+    '../projects/canopy/src/lib/side-nav/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/side-nav/docs/side-nav.stories.ts',
+    // Templates
+    '../projects/canopy/src/lib/page/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/page/docs/page.stories.ts',
+    // Helpers
+    //  Directives
+    '../projects/canopy/src/lib/feature-toggle/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/feature-toggle/docs/feature-toggle.stories.ts',
+    '../projects/canopy/src/lib/focus/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/focus/docs/focus.stories.ts',
+    '../projects/canopy/src/lib/grid/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/grid/docs/grid.stories.ts',
+    '../projects/canopy/src/lib/hide-at/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/hide-at/docs/hide-at.stories.ts',
+    '../projects/canopy/src/lib/shadow/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/shadow/docs/shadow.stories.ts',
+    '../projects/canopy/src/lib/show-at/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/show-at/docs/show-at.stories.ts',
+    '../projects/canopy/src/lib/skeleton/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/skeleton/docs/skeleton.stories.ts',
+    '../projects/canopy/src/lib/spacing/margin/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/spacing/margin/docs/margin.stories.ts',
+    '../projects/canopy/src/lib/spacing/padding/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/spacing/padding/docs/padding.stories.ts',
+    '../projects/canopy/src/lib/spacing/row-gap/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/spacing/row-gap/docs/row-gap.stories.ts',
+    '../projects/canopy/src/lib/sr-alert-message/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/sr-alert-message/docs/sr-alert-message.stories.ts',
+    '../projects/canopy/src/lib/variant/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/variant/docs/variant.stories.ts',
+    //  Pipes
+    '../projects/canopy/src/lib/pipes/camel-case/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/pipes/camel-case/docs/camel-case.stories.ts',
+    '../projects/canopy/src/lib/pipes/kebab-case/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/pipes/kebab-case/docs/kebab-case.stories.ts',
+    //  Components
+    '../projects/canopy/src/lib/heading/docs/guide.stories.mdx',
+    '../projects/canopy/src/lib/heading/docs/heading.stories.ts',
+    //  Style
+    '../projects/canopy/src/styles/docs/mixins.stories.mdx',
+    '../projects/canopy/src/styles/docs/utils.stories.mdx',
+
+
   ],
   addons: [
     '@storybook/addon-a11y',
     '@storybook/addon-essentials',
+    {
+      name: "@storybook/addon-docs",
+      options: { transcludeMarkdown: true },
```

Comment:
This option is to allow to import markdown files in an `mdx` file using the tag, for example:

```
import MyMarkdown from 'my-markdown.md';

# Some mdx here

<MyMarkdown />
```